packit/upstream.py

```diff
@@ -232,7 +231,7 @@ def create_patches(
         upstream = upstream or self.get_specfile_version()
         destination = destination or self.local_project.working_dir
 
-        commits = self.get_commits_to_upstream(upstream, add_upstream_head_commit=True)
+        commits = self.get_commits_to_upstream(upstream, add_upstream_head_commit=False)
```

Comment:
Why is the list of coomits needed here? `git format-patch` can create a list of patches in a given revision range.

Reply:
Maybe because previously (68088a2456) we were running `git diff` for each commit to generate the patches.